### PR TITLE
Hide command bar from CompositeList when no commands exist

### PR DESCRIPTION
**Pull request by chankev**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Hide command bar from CompositeList if no commands exist

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7885)

### CODE REVIEW

File: packages/dashboard/src/components/CompositeList/CompositeList.styles.ts [hunk 1]
```diff
@@ -88,17 +88,17 @@ export const getCommandBarSelectionCountItemStyles = (): Partial<IButtonStyles>
   };
 };
 
-export const getDetailsListStyle = (): IStyle => {
+export const getDetailsListStyle = (showCommandBar: boolean): IStyle => {
```
Comment (natalieethell) [inline packages/dashboard/src/components/CompositeList/CompositeList.styles.ts]: Should this be optional for backwards compatibility?

**Comment by natalieethell** [comment]
I'll look into the snapshot failures. They're unrelated to your change.

**Comment by natalieethell** [comment]
Once this PR is merged, your build should succeed: https://github.com/OfficeDev/office-ui-fabric-react/pull/7896.

File: packages/dashboard/src/components/CompositeList/CompositeList.styles.ts (hunk 1 above)
Comment (chankev) [reply packages/dashboard/src/components/CompositeList/CompositeList.styles.ts]: I have updated it to be an optional prop.

**Pull request merged**